src/RockBridge.cpp

```diff
@@ -62,6 +69,11 @@ void RockBridge::Load(int _argc , char** _argv)
     RTT::corba::TaskContextServer::ThreadOrb(ORO_SCHED_OTHER, RTT::os::LowestPriority, 0);
 
     // Import typekits to allow RTT convert the types used by the components
+    RTT::types::TypekitRepository::Import(new orogen_typekits::rock_gazeboTypekitPlugin);
```

Comment:
Any reason why you moved that stuff up there ? Seemed more logical to me to load first std/base (even though it probably makes no difference, it's just how it looks)